examples/consumer_rebalance_example/consumer_rebalance_example.go

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Example high-level Apache Kafka consumer demonstrating use of rebalance
+// callback along with manual commit.
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+)
+
+func main() {
+	if len(os.Args) < 4 {
+		fmt.Fprintf(os.Stderr, "Usage: %s <bootstrap-servers> <group> <topics..>\n",
+			os.Args[0])
+		os.Exit(1)
+	}
+
+	var err error
+	bootstrapServers := os.Args[1]
+	group := os.Args[2]
+	topics := os.Args[3:]
+
+	sigchan := make(chan os.Signal, 1)
+	signal.Notify(sigchan, os.Interrupt)
+
+	c, err := kafka.NewConsumer(&kafka.ConfigMap{
+		"bootstrap.servers": bootstrapServers,
+		// Consumer group ID
+		"group.id": group,
+		// Start reading from the first message of each assigned
+		// partition if there are no previously committed offsets
+		// for this group.
+		"auto.offset.reset": "earliest",
+		// Whether or not we commit offsets automatically.
+		"enable.auto.commit": false,
+	})
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to create consumer: %s\n", err)
+		os.Exit(1)
+	}
+
+	fmt.Printf("%% Created Consumer %v\n", c)
+
+	// Subscribe to topics, call the rebalanceCallback on assignment/revoke.
+	// The rebalanceCallback can be triggered from c.Poll() and c.Close().
+	err = c.SubscribeTopics(topics, rebalanceCallback)
+
+	run := true
+	for run == true {
+		select {
+		case sig := <-sigchan:
+			fmt.Printf("%% Caught signal %v: terminating\n", sig)
+			run = false
+		default:
+			ev := c.Poll(100)
+			if ev == nil {
+				continue
+			}
+
+			if err = processEvent(c, ev); err != nil {
+				fmt.Fprintf(os.Stderr, "Failed to process event: %s\n", err)
```

Comment:
I'd include the event too, e.g:
```suggestion
				fmt.Fprintf(os.Stderr, "Failed to process event: %v: %s\n", ev, err)
```